textproc/linuxdoc-tools/Portfile

```diff
@@ -1,41 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem 1.0
 
 name            linuxdoc-tools
-version         0.9.67
+version         0.9.73
+checksums       rmd160  1a168312d1a947a3a11f4ae6e6631f33695feb0c \
+                sha256  6cc169c350ba40183714f7e42ff8f06da5ae151603fcaf7b806bf0284b66dc75 \
+                size    518277
+
```

Comment:
It's a bit unusual stylistically to have the checksums all the way up at the top, but you have more experience than me.